pkg/apollo/server.go

```diff
@@ -31,18 +31,35 @@ const localRecordsPath = "./data/local.json"
 
 var data []schema.Record
 
+func enableCors(w *http.ResponseWriter) {
+	(*w).Header().Set("Access-Control-Allow-Origin", "*")
+}
+
+func setupResponse(w *http.ResponseWriter, req *http.Request) {
```

Comment:
@dborowiec10 `http.ResponseWriter` is an interface. Unfortunately I don't have Apollo setup yet and couldn't find the article explaining why pointers to interfaces don't make sense. 😏

This is the next best thing I found: https://stackoverflow.com/questions/13255907/in-go-http-handlers-why-is-the-responsewriter-a-value-but-the-request-a-pointer
In short: `w` is already a pointer to a struct which is just hidden by the interface.